frontend/src/component/menu/Header/Header.tsx

```diff
@@ -122,9 +123,16 @@ const StyledLinkWithBetaBadge = ({
     to,
 }: { title: string; to: string }) => (
     <StyledLink to={to} sx={{ margin: 0 }}>
-        <div>
+        <Typography
+            component='div'
+            sx={(theme) => ({
+                display: 'flex',
+                alignItems: 'center',
+                gap: theme.spacing(1),
+            })}
```

Comment:
Don't we recommend not using the `sx` prop, instead preferring styled components (for performance reasons, I think)?

If so, I think we should just quickly drag this out into a separate component to do just that.

Reply:
No worries, changed to styled component